tools/parser_sql/processors/AbstractProcessor.php

```diff
@@ -157,7 +157,7 @@ protected function isWhitespaceToken($token) {
     }
 
     protected function isCommentToken($token) {
-        return isset($token[0]) && isset($token[1])
+        return isset($token[0], $token[1])  
```

Comment:
Isn't CS fixer saying anything about the additional whitespaces?

Reply:
maybe it was not run?

Reply:
Sorry, I did not run php-cs-fixer, but I will ❤️